app/src/main/java/org/mozilla/focus/browser/cursor/CursorViewModel.kt

```diff
@@ -43,7 +43,7 @@ private const val DOWN_TIME_OFFSET_MILLIS = 100
  * @param simulateTouchEvent Takes the given touch event and simulates a touch to the screen.
  */
 class CursorViewModel(
-        private val onUpdate: (x: Float, y: Float, scrollVel: PointF) -> Unit,
+        private val onUpdate: (x: Float, y: Float, percentMaxScrollVel: PointF, framesPassed: Float) -> Unit,
```

Comment:
nit: rename `framesPassed` too to match multiplier from earlier.

Reply:
What is the difference between `onUpdate` and `updatePosForVel`?

Reply:
> nit: rename framesPassed too to match multiplier from earlier.

@liuche I don't understand, what do you mean?

Reply:
> What is the difference between onUpdate and updatePosForVel?

- `updatePosForVel` is a helper function mutates the current object to 
- `onUpdate` is a callback to inform listeners that the position has changed. `on*` is conventionally for callbacks like this in Java.